### DISCUSSION TITLE
Get mesh from GLTF scene

### DISCUSSION THREAD

**Discussion by IllyStable** [bevyengine/bevy#13550] [Q&A] [answered]
Hi,

I'm working on a collision system for a game I'm working on, where the map is loaded in from a GLTF file, however, my collision system uses the Mesh datatype - is there anyway to convert between the two?

**Comment by GitGhillie** [comment, marked as answer]
When you load a glTF through spawning SceneBundle it will create an hierarchy of entities, and somewhere in there will be an entity with a Handle<Mesh> of the mesh that you want to use for collisions.

To get an idea of what the hierarchy looks like so you can make the correct query, I'd recommend looking at it with https://github.com/jakobhellermann/bevy-inspector-egui?tab=readme-ov-file#worldinspectorplugin
